src/guide/events.md

```diff
@@ -1,5 +1,7 @@
 # Event Handling
 
+<VideoLesson href="https://vueschool.io/lessons/user-events-in-vue-3?friend=vuejs" title="Learn how to handle events on Vue School">Learn how to handle events in a free Vue School lesson</VideoLesson>
```

Comment:
The lesson doesn't start in on the event material till halfway through. Not sure this is a good way to introduce people to a concept.